### PR TITLE
Missing data hover info

### LINKED ISSUE #368: Add "No estimate" hover to casemap
https://epidemicforecasting.slack.com/archives/C0100BY1EJZ/p1586198610000100

### PR DESCRIPTION
**Pull request by veselyvojtech**
Closes #368

### CODE REVIEW

**Comment by veselyvojtech** [comment]
I believe there could be done a lot more optimization, e.g. with loading the geojson. I didn't know how to do that it is not read twice. And I believe also on couple of other stuff.

File: frontend/case-map.ts [hunk 1]
```diff
@@ -42,54 +35,120 @@ function makeMap(regions_data) {
...
-      return dict[country]["iso_alpha_3"];
-    });
+  const iso_key = "iso_a3";
+
+  var countries_json = JSON.parse(
+    $.ajax({
+      url:
+        "https://storage.googleapis.com/static-covid/static/casemap-geo.json",
+      dataType: "json",
+      async: false
```
Comment (mathijshenquet) [inline frontend/case-map.ts:45]: Please don't use blocking requests

Comment (mathijshenquet) [reply frontend/case-map.ts:45]: Its better to add this to L210 and then pass it as an argumen to makeMap. There is something similar in `lines.ts`.

File: frontend/case-map.ts [hunk 2]
```diff
@@ -42,54 +35,120 @@ function makeMap(regions_data) {
     return x.toString();
   }
 
-  function get_locations(dict) {
-    return Object.keys(dict).map(function(country) {
-      return dict[country]["iso_alpha_3"];
-    });
+  const iso_key = "iso_a3";
```
Comment (mathijshenquet) [inline frontend/case-map.ts:38]: Could you make this constant uppercase?

File: frontend/case-map.ts (hunk 1 above)
Comment (veselyvojtech) [reply frontend/case-map.ts:45]: Sorry, I spent a lot of time trying to load the json content to some variable, but I couldn't make it work without that `async: false` option. And I still don't know how to do that.

**Comment by mathijshenquet** [comment]
@veselyvojtech I could quickly fix these issues. Would this be ok?

**Comment by veselyvojtech** [comment]
@mathijshenquet Hi, thanks, that would be great and it is ok with me!

**Pull request merged**